js/ui/control/scale.js

```diff
@@ -12,8 +12,10 @@ module.exports = Scale;
  * @class Scale
  * @param {Object} [options]
  * @param {string} [options.position='bottom-left'] A string indicating the control's position on the map. Options are `'top-right'`, `'top-left'`, `'bottom-right'`, and `'bottom-left'`.
+ * @param {integer} [options.maxWidth='150'] A number indicating the maximum length of the scale control in pixels.
```

Comment:
`integer` -> `number`; you could also remove `A number indicating ` and preserve the meaning fully